src/components/program-progress/ProgramListingCard.jsx

```diff
@@ -0,0 +1,138 @@
+import { breakpoints, Card } from '@edx/paragon';
+import React, { useContext, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useHistory } from 'react-router-dom';
+
+import { AppContext } from '@edx/frontend-platform/react';
+
+const ProgramListingCard = ({ program }) => {
+  const { enterpriseConfig } = useContext(AppContext);
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth);
+  const history = useHistory();
+
+  const handleCardClick = () => {
+    history.push(`/${enterpriseConfig.slug}/program-progress/${program.uuid}`);
+  };
+
+  useEffect(() => {
+    const handleWindowResize = () => {
```

Comment:
What is the purpose of this code?

Reply:
when window is resized, it updates the state variable `windowWidth` present in the scope, which in turn is being used to decide which image should be rendered.